app/uk/gov/hmrc/apidocumentation/models/Documentation.scala

```diff
@@ -54,6 +54,7 @@ case class APIDefinition(
   lazy val statusSortedActiveVersions = statusSortedVersions.filterNot(v => v.status == APIStatus.RETIRED)
   lazy val defaultVersion = statusSortedActiveVersions.headOption
   lazy val hasActiveVersions = statusSortedActiveVersions.nonEmpty
+  lazy val isRestOrXmlApi = Seq(REST_API, XML_API).exists(_ == label)
   lazy val label: APIDefinitionLabel =
```

Comment:
This could be `Seq(REST_API, XML_API).contains(label)` or `label == REST_API || label == XML_API`. I think either is more readable.